src/dotnet-serve/dotnet-serve.csproj

```diff
@@ -23,6 +23,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" Publish="false"/>
     <PackageReference Include="McMaster.Extensions.CommandLineUtils" Version="3.0.0" />
+    <PackageReference Include="dotnet-config-lib" Version="1.0.0-alpha.6" />
```

Comment:
How stable is this new dependency? Do you expect there will be more changes before this is considered "RTM"?

Reply:
I've been fixing minor things but the API is largely stable at this point. I think it would be best to wait until it's 1.0.0 before merging this (which should happen soon-ish). The only pending thing is to exercise it with a few more consumers and call it stable (got a bunch in line already).